src/libOpenImageIO/maketexture.cpp

```diff
@@ -931,7 +1034,20 @@ make_texture_impl (ImageBufAlgo::MakeTextureMode mode,
         mode = ImageBufAlgo::MakeTxEnvLatl;
         src = latlong;
     }
-
+    
+    if (mode == ImageBufAlgo::MakeTxBumpWithSlopes ){
+        ImageSpec newspec = src->spec();
```

Comment:
This is probably the right place to check and issue errors for any invalid inputs, like unexpected number of channels in the bump image, etc.